docs/charts/radar.md

```diff
@@ -152,12 +152,17 @@ The radar chart defines the following configuration options. These options are m
 ## Scale Options
 
 The radar chart supports only a single scale. The options for this scale are defined in the `scale` property.
+The options for `scale` property can be refererenced from the [Linear Radial Axis page](../axes/radial/linear).
```

Comment:
How about rewording to:
> The options for this scale are defined in the `scale` property, which can be referenced from the [Linear Radial Axis page](../axes/radial/linear)

("referenced" is spelled wrong, so make sure to update to the spelling I used)